tests/e2e/tests/pilot/testdata/networking/v1alpha3/wildcard-tls-google-no-vs.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: wildcardgoogletls
+spec:
+  hosts:
+  - "*.google.co.uk"
+  - "*.google.co.in"
+  endpoints:
+  - address: google.co.uk
```

Comment:
skip

Reply:
this one tests TLS without VS.

Reply:
So TLS without VS should work now, shouldn't it?

Reply:
yes it should!  so is https without VS.